hbase2/src/test/java/site/ycsb/db/hbase2/HBaseClient2Test.java

```diff
@@ -15,6 +15,8 @@
 
 package site.ycsb.db.hbase2;
 
+import static org.apache.hadoop.util.StringUtils.hexStringToByte;
```

Comment:
use `org.apache.hadoop.hbase.util.Bytes.fromHex` instead since it is part of HBase's public API.